### DISCUSSION TITLE
Spiral Linux vs Emmabuntüs linux

### DISCUSSION THREAD

**Discussion by arky217** [SpiralLinux/SpiralLinux-project#512] [General]
Spiral linux and Emmabuntüs linux seem to be fulfilling the same role.
Are there any major differences ?

**Comment by geckolinux** [comment]
Hi there, apart from being based on Debian and using the Calamares installer they seem to have little in common.
- https://emmabuntus.org/emmabuntus-de5/
- https://spirallinux.github.io/#spirallinux-features
